bodhi/client/__init__.py

```diff
@@ -95,6 +95,14 @@ def _warn_if_url_and_staging_set(ctx, param, value):
     staging_option,
     url_option]
 
+# Basic options for pagination of query result
+pagination_options = [
+    click.option('--rows', default=None,
+                 type=click.IntRange(1, 100, clamp=True),
```

Comment:
I think it could be confusing if we silently clamp larger values to ```100``` here. Users may try to do ```--rows 200``` and may not notice they are getting 100 back, and then may page to page 2 skipping 100 items. Let's use ```clamp=False``` so that users are aware when they exceed the page size.